.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,18 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
```

Comment:
Yes, how our release tags gonna look? Or maybe I will change to release instead of tags.

Reply:
We haven't decided yet, but I think the way you have it now makes sense.

Reply:
I change it to github release event thats better.